### ISSUE TITLE
Cannot use 'in' operator to search for 'stringify' in null

### ISSUE THREAD

**Issue by creage** [perchlayer/po-loader#7]
```
Module build failed: TypeError: Cannot use 'in' operator to search for 'stringify' in null
    at Object.module.exports (C:\some_project\node_modules\po-loader\index.js:14:20)
```

node 8.1.2
webpack 2.6.1

**Comment by dschissler** [comment]
Thanks.  Apparently getOptions returns null when there are no options.  For the moment you can drop down to version 0.3.*.

**Comment by dschissler** [comment]
Fixed in 0.4.1
